tools/clang/unittests/SPIRV/FileTestFixture.cpp

```diff
@@ -0,0 +1,129 @@
+//===- FileTestFixture.cpp ------------- File Test Fixture Implementation -===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include <fstream>
+
+#include "FileTestFixture.h"
+#include "FileTestUtils.h"
+
+#include "effcee/effcee.h"
+
+namespace clang {
+namespace spirv {
+
+namespace {
+const char hlslStartLabel[] = "// Run:";
+const std::vector<std::string> checkCommands = {"CHECK:",       "CHECK-NEXT:",
```

Comment:
I think `const char*[]` will suffice? Non-POD global variables can cause trouble. Actually Google C++ style guide forbids them completely. Although we are not following Google C++ style guide in LLVM project. Still good to keep in mind. https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables